react-components/jsx.md

```diff
@@ -182,59 +184,70 @@ In this situation, the rendered HTML would look like:
 
 ### Embedding Expressions: `App.js`
 
-Let's revisit this `src/App.js` again!
+Let's revisit this `src/App.jsx` again!
 
 <!-- prettier-ignore-start -->
 ```js
-import logo from './logo.svg';
-import './App.css';
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
 
 function App() {
+  const [count, setCount] = useState(0)
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+    <>
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src={viteLogo} className="logo" alt="Vite logo" />
+        </a>
+        <a href="https://react.dev" target="_blank">
+          <img src={reactLogo} className="logo react" alt="React logo" />
+        </a>
+      </div>
+      <h1>Vite + React</h1>
+      <div className="card">
+        <button onClick={() => setCount((count) => count + 1)}>
+          count is {count}
+        </button>
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Edit <code>src/App.jsx</code> and save to test HMR
         </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+      </div>
+      <p className="read-the-docs">
+        Click on the Vite and React logos to learn more
+      </p>
+    </>
+  )
 }
+
+export default App
 ```
 <!-- prettier-ignore-end -->
 
 We can focus on this part of the JSX:
 
 ```js
-<img src={logo} className="App-logo" alt="logo" />
+<img src={viteLogo} className="logo" alt="Vite logo" />
 ```
 
-The value of the `src` attribute for this element will be the value of `logo`. `logo` was imported from `./logo.svg` at the top of `App.js`!
+The value of the `src` attribute for this element will be the value of `viteLogo`. `viteLogo` was imported from `/vite.svg` at the top of `App.jsx`!
```

Comment:
OMG. Vite tempalte craziness strikes again. Why is it using an absolute path to the SVG. And why isn't the SVG actually in the root of the project. Oh. That's right. BECAUSE IT'S IN `public`! 🤦 

We should mention where the file is actually resolved to. Or if we want to avoid that, swicth to discussing `reactLogo` and hope no one notices the vite logo strangeness.